docs/installation/index.md

```diff
@@ -1,33 +1,49 @@
 <!--[metadata]>
 +++
-title = "Supported installation"
+title = "Supported installations for Docker Engine"
 description = "Lists the installation methods"
 keywords = ["Docker install "]
 +++
 <![end-metadata]-->
 
-# Supported installation
+# Install Docker Engine 
 
-Docker supports installation on the following: 
+Docker Engine is supported on Linux, Cloud, Windows, and OS X. Installation instructions are available for the following: 
 
-* [Amazon EC2 Installation](amazon)
+## On Linux
 * [Arch Linux](archlinux)
-* [Microsoft Azure platform](azure)
-* [Installation from binaries](binaries)
 * [CentOS](centos)
 * [CRUX Linux](cruxlinux)
 * [Debian](debian)
 * [Fedora](fedora)
 * [FrugalWare](frugalware)
 * [Gentoo](gentoolinux)
-* [Google Cloud Platform](google)
-* [Install on Joyent Public Cloud](joyent)
-* [Mac OS X](mac)
 * [Oracle Linux](oracle)
-* [Rackspace Cloud](rackspace)
 * [Red Hat Enterprise Linux](rhel)
-* [IBM SoftLayer](softlayer)
 * [openSUSE and SUSE Linux Enterprise](SUSE)
 * [Ubuntu](ubuntulinux)
+
+## Other Linux
```

Comment:
@mikebrow Thanks for doing this.  Looks great.  I'd remove this heading `## Other Linux`